client/app/components/pages/workspace/annotation.component.ts

```diff
@@ -63,16 +91,42 @@ export class AnnotationComponent implements OnInit {
 		this.currentPointInDraw = null;
 		this.annotations = [];
 		this.displayedAnnotations = [];
+		this.annoText = "Enter text for line"
+		this.lines = this.pageAnnotation.freeDraws;
+		this.annoNumber = 1;	
+		if(this.pageAnnotation){
+			this.lines = this.pageAnnotation.freeDraws
+		}
+		if (this.lines.length == 0){
+			this.addFreeDrawAnno()
+		}
+		this.currLine = this.lines[0]
+	}
+	addFreeDrawAnno(){
```

Comment:
I think that after creating a new line the current line should change to it.
what you think?